lib/interpolateName.js

```diff
@@ -3,11 +3,16 @@
 const path = require('path');
 const emojisList = require('emojis-list');
 const getHashDigest = require('./getHashDigest');
+const { WordUglify, FileUglify, CssUglify } = require('uglify-set');
 
 const emojiRegex = /[\uD800-\uDFFF]./;
 const emojiList = emojisList.filter((emoji) => emojiRegex.test(emoji));
 const emojiCache = {};
 
+const wordUglify = WordUglify();
+const fileUglify = FileUglify();
+const cssUglify = CssUglify();
```

Comment:
Potential memory leak in watch mode

Reply:
Oh you're right
Is there a way to detect it?

Reply:
It seems memory leak can fire anywhere not only here.
To solve this I'm adding forcing reset code in `uglify-set`.
But there is no specification maximum size of string in Ecma.
We need some standard, if you have some idea plz tell me :)
I'll also research about this.

Reply:
http://www.ecma-international.org/ecma-262/6.0/#sec-array-exotic-objects
According to spec, `length` of Array is less than 2^32.
How about follow this?

Reply:
only weak cache can solve this problem

Reply:
Maybe I'm not catch the point now.
Can you explain it more detail?